keyloop/ext/sqla/models.py

```diff
@@ -1,15 +1,46 @@
+import sqlalchemy as sa
+import transaction
 from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.orm import relationship, backref
 from zope.interface import implementer
-import sqlalchemy as sa
 
+from keyloop.interfaces.identity import IIdentity, IIdentitySource, IContact
 from keyloop.ext.utils.decorators import singleton, singletonmethod
-from keyloop.interfaces.identity import IIdentity, IIdentitySource
 
 
-@implementer(IIdentity)
-class Identity:
+@implementer(IContact)
+class Contact:
+    __tablename__ = "contact"
+
+    @declared_attr
+    def id(self):
+        return sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    @declared_attr
+    def type(self):
+        return sa.Column(sa.String, index=True)
+
+    @declared_attr
+    def value(self):
+        return sa.Column(sa.String, index=True)
+
+    @declared_attr
+    def valid_for_auth(self):
+        return sa.Column(sa.Boolean, default=False)
+
+    @declared_attr
+    def identity_id(self):
+        return sa.Column(sa.Integer, sa.ForeignKey('identity.id'))
+
+    # TODO: Make this attribute available on response schema
+    def contacts(self):
+        return relationship('Identity', backref=backref('contact', lazy='dynamic'), foreign_keys=[self.identity_id])
```

Comment:
```suggestion
        return relationship('Identity', backref=backref('contacts', lazy='dynamic'), foreign_keys=[self.identity_id])
```

Além disso, precisamos entender como fazer o `relationship` do sqalchemy funcionar no contexto do `declared_attr`.